src/components/PCInput.jsx

```diff
@@ -19,33 +21,52 @@ class PCInput extends Component {
 		let value = event.target.value
 		this.setState({
 			inputVal: value
-		})
+		}, () => {
+			this.props.lastPseudo(this.state.inputVal)
+		});
 	}
 
 	handleSubmission(event){
-		let pseudo = event.target.value;
+		hotkeys('ctrl+/,ctrl+.', function(e,handler) {
+		  console.log(e,handler);
+		  switch(handler.key){
+		    case "ctrl+/":alert('create child!');break;
+		    case "ctrl+.":alert('back to parent!');break;
+		  }
+		});
+
+		// let pseudo = event.target.value;
+		let pseudo = this.state.inputVal;
+		let interPseudo = this.props.pseudo;
 		let key = event.key;
 		let index = this.props.pseudoIndex;
 
 		if (key === 'Enter'){
-			if (this.props.pseudo){
-				this.props.editPseudo(index, pseudo)
+			if (interPseudo){
+				this.props.editPseudo(index, interPseudo)
 			} else {
 				this.props.addPseudo(pseudo);
 				event.target.value = '';
 			}
 
 		}
+		if (key === 'Tab'){
```

Comment:
Going forward, don't forget we're going to need to handle combinations of keys; "Tab" already has expected behavior we don't want to override for accessibility reasons

Reply:
I installed a hotkey package and played with it alittle in pcinput, and decided to move on to crud

Reply:
Crud is finito. I am going to restore the scope of your component files, and get back on the hotkeying. After that, I will like to follow up with you on the phone whenever you have time on just shooting ideas on how the "structure flow" of how parents and childs are created